src/components/ha-svg-icon.ts

```diff
@@ -40,6 +51,14 @@ export class HaSvgIcon extends LitElement {
         pointer-events: none;
         display: block;
       }
+      path.primary-path {
+        fill: var(--icon-primary-color, currentcolor);
```

Comment:
I would move this to `:host` to make sure we are not breaking anything (as the `fill` in `host:` will not longer be used and no longer be possible to override)